frontend/src/component/ui/Toggle.spec.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Toggle } from './Toggle';
+
+describe('Toggle component', () => {
+  test('should render without crash', () => {
+    render(<Toggle />);
+  });
+
+  test('should render an inactive toggle when disabled by default', () => {
+    const { container } = render(<Toggle defaultIsEnabled={false} />);
+
+    expect(container.querySelector('.background')).toBeTruthy();
+    expect(container.querySelector('.knob')).toBeTruthy();
+    expect(container.querySelector('.background.active')).toBeNull();
+    expect(container.querySelector('.knob.active')).toBeNull();
+  });
+
+  test('should render an active toggle when enabled by default', () => {
+    const { container } = render(<Toggle defaultIsEnabled={true} />);
+
+    expect(container.querySelector('.background.active')).toBeTruthy();
+    expect(container.querySelector('.knob.active')).toBeTruthy();
+  });
+
+  test('should toggle from disabled to enabled', () => {
+    const toggleRef = React.createRef<Toggle>();
+    const { container } = render(
+      <Toggle ref={toggleRef} defaultIsEnabled={false} />
+    );
+
+    expect(container.querySelector('.background')).toBeTruthy();
+    expect(container.querySelector('.knob')).toBeTruthy();
+    expect(container.querySelector('.background.active')).toBeNull();
+    expect(container.querySelector('.knob.active')).toBeNull();
+
+    toggleRef.current?.handleClick();
```

Comment:
Can we use `fireEvent.click()` like we did in [here](https://github.com/short-d/short/blob/0cf77bb3598d7eec258dc5790097aa86e33562d0/frontend/src/component/ui/Icon.spec.tsx#L29)?
Otherwise, the tests will pass even when the `onClick` attribute of the div containing the toggle is modified.

Reply:
Please change everything to fireEvent.click as suggested!